fvm/evm/evm_test.go

```diff
@@ -1513,11 +1512,13 @@ func TestDryRun(t *testing.T) {
 					evmAddress,
 				))
 
+				// Use the gas estimation from Evm.dryRun with some buffer
+				gasLimit := dryRunResult.GasConsumed + gethParams.SstoreSentryGasEIP2200
```

Comment:
Any specific reason to use `SstoreSentryGasEIP2200` as buffer?

Reply:
Here as well, the buffer gas is required to be present in the gas limit.
This case is testing the scenario: `test dry run store current value`.
The description of `SstoreSentryGasEIP2200` is the following:
```bash
Minimum gas required to be present for an SSTORE call, not consumed
```

I think this is something special for avoiding re-entrancy attacks.